.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest a new algorithm, enhancement to an existing algorithm, etc.
+title: ''
+labels: Enhancement, New Feature
```

Comment:
```suggestion
labels: New Feature
```

We can always relabel later to Enhancement rather than a New Feature but it's rarely both at the same time.